content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -193,19 +163,59 @@ to manage the lifecycle of Istio.
     58
     {{< /text >}}
 
-1. Install the `istio` chart:
+1. Install the `istio` chart. For a production environment use the recommended Helm values or customize as needed:
 
     {{< text bash >}}
-
     $ helm install istio.io/istio --name istio --namespace istio-system
+    {{< /text >}}
+
+    {{< tip >}}
```

Comment:
Why is this a tip instead of the last step? Please make both instances into steps to end the procedure for readers with sense of accomplishment.